### PR TITLE
Implement Tag List View for Ui

### PR DESCRIPTION
**Pull request by Ma-Yueran**
Tags and addresses of files can be displayed now.

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -28,27 +28,30 @@
     private final Logger logger = LogsCenter.getLogger(getClass());
 
     private Stage primaryStage;
+    private ResultDisplay resultDisplay;
+    private TagListPanel tagListPanel;
+    private LastInputDisplay lastInputDisplay;
```
Comment (luo-git) [inline src/main/java/seedu/address/ui/MainWindow.java]: Are these 3 variables no longer independent Ui parts?

`// Independent Ui parts residing in this Ui container`

**Review by luo-git — APPROVED** [review]
Looks good to merge. Did you refactor new Ui as MainApp?

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (Ma-Yueran) [reply src/main/java/seedu/address/ui/MainWindow.java]: I think I put them in a wrong place while modifying, did not notice that

**Review by Ma-Yueran — COMMENTED** [review]
> Looks good to merge. Did you refactor new Ui as MainApp?

Yeah, MainApp -> OldMainApp, NewMainApp -> MainApp

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (Ma-Yueran) [reply src/main/java/seedu/address/ui/MainWindow.java]: I updated the PR to move them under the "independent Ui part" comment. Thank you for pointing that out.

**Pull request merged**